charts/opentelemetry-demo/Chart.yaml

```diff
@@ -11,22 +11,22 @@ maintainers:
   - name: puckpuck
   - name: tylerhelmuth
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: "1.10.0"
+appVersion: "1.11.0"
 dependencies:
   - name: opentelemetry-collector
-    version: 0.93.1
+    version: 0.96.0
     repository: https://open-telemetry.github.io/opentelemetry-helm-charts
     condition: opentelemetry-collector.enabled
   - name: jaeger
-    version: 3.0.10
+    version: 3.1.0
     repository: https://jaegertracing.github.io/helm-charts
     condition: jaeger.enabled
   - name: prometheus
-    version: 25.21.0
+    version: 25.22.0
     repository: https://prometheus-community.github.io/helm-charts
     condition: prometheus.enabled
   - name: grafana
-    version: 7.3.11
+    version: 8.2.0
```

Comment:
This does a major version of Grafana that has not been tested yet on the demo itself. I think we should hold back this bump until after we make sure Grafana 11 is fine with the demo.